components/help-queue/help-request-search.tsx

```
@@ -60,9 +60,10 @@ const getStatusColor = (status: string) => {
 /**
  * Simple component to display student name from profile ID
  */
-function StudentName({ profileId }: { profileId: string }) {
+function StudentName({ profileId, fallbackCreatedBy }: { profileId: string; fallbackCreatedBy?: string }) {
   const profile = useUserProfile(profileId);
-  return <>{profile?.name || "Unknown Student"}</>;
+  const fallbackProfile = fallbackCreatedBy ? useUserProfile(fallbackCreatedBy) : null;
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)